packages/grid/scss/_css-grid.scss

```diff
@@ -75,6 +76,7 @@
         }
 
         @include breakpoint($key) {
+          /* stylelint-disable-next-line max-nesting-depth */
```

Comment:
Shouldn't this be covered by the exceptions for `@` rules?

Reply:
added this because here the nesting-depth exceeded 3 ->  but as mentioned this isn't required anymore